### ISSUE TITLE
GPL-881 As a sample management manager I would like to be able to flag which boxes are surplus to requirements [Heron]

### ISSUE THREAD

**Issue by stevieing** [sanger/labwhere#322]
**User story**
As a sample management manager I would like to be able to flag which boxes are surplus to requirements

priority: 4

**Who are the primary contacts for this story**
Danni / Bart

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] A page where I would be able to mark boxes for moving to reefers
- [ ] A page where I would be able to find which boxes are available to be moved

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by rl15** [comment]
Bart Siwek wrote (Monday, 15 March 2021 at 15:27)

I don’t think this is required, as we treat anything that is not protected as surplus to requirements:

**Comment by stevieing** [comment]
Closed as no longer needed.
